_includes/tutorials/console-consumer-producer-avro/kafka/markup/dev/create-schema.adoc

```diff
@@ -0,0 +1,7 @@
+Next let’s create the schema to use when formatting the messages to produce to the topic created in previous step.
+
+From the same terminal you used to create the topic, create the following `order_detail.avsc` file inside the directory `schema`.
```

Comment:
I think a user might get tripped here, how about we break this into two steps:

`mkdir schema`

and then
> create a new file named `order_detail.avsc` inside the new `schema` folder with the following contents.

Reply:
I see that you're creating the volume mount now so that it's available to cli commands inside the containers.  I got tripped here, so maybe changed the wording slightly:

```suggestion
From the same terminal you used to create the topic, create the following `order_detail.avsc` file inside the existing `schema` folder.
```